starter-code/iteration-3/main.ts

```diff
@@ -1,9 +1,55 @@
 // Add the reference to the interface
+import { IToDoItem, IToDo} from './interfaces';
 
 // Create class TodoItem that implements the corresponding interface
 
+class TodoItem implements IToDoItem {
+  status: boolean;
+  updatedAt: number;
+
+
+
+    constructor (public title:string) {
+      this.updatedAt = Date.now();
+      this.status    = false;
+    }
+
+    toggleStatus():void {
+      this.status = !this.status;
+      this.updatedAt = Date.now();
+    }
+
+}
+
+
 // Create class TodoList that implements the corresponding interface
 
+
+class TodoList implements IToDo {
+  ToDoItems: Array <object> = [];
+
+
+  addTask(theItem:ToDoItem):number {
+    console.log(TodoItem.title)
+    this.ToDoItems.push(TodoItem);
+    console.log(outputString);
+    return this.ToDoItems.length
+    }
+
+    listAllTasks():void {
+      console.log("======ALL TASK=====")
+      this.ToDoItems.forEach((task) => { console.log(task.title)});
```

Comment:
You would get an error on "title" since you forgot to include it as a type in your object.